engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/db/PeriodicProcessesRepository.scala

```diff
@@ -112,6 +112,10 @@ trait PeriodicProcessesRepository {
 
   def markInactive(processId: PeriodicProcessId): Action[Unit]
 
+  def getSchedulesState(
+      processName: ProcessName
```

Comment:
`scenarioName` (we agreed to not use the "Process" word anymore - `ProcessName` will be refactored in the future).